src/get-started/index.njk

```diff
@@ -0,0 +1,237 @@
+</p>
+
+<h2 class="govuk-heading-l" id="Patterns-components-and-styles">
+  Patterns, components and styles
+</h2>
+
+<p class="govuk-body">
+All designs for HMRC services must follow <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK principles</a> and use <a href="https://design.tax.service.gov.uk/hmrc-design-patterns/" class="govuk-link">HMRC</a> and <a href="https://www.gov.uk/guidance/government-design-principles" class="govuk-link">GOV.UK</a>
+design patterns, components and styles. 
+</p>
+
+<p class="govuk-body">
+If a pattern exists in both HMRC and GOV.UK design libraries use the HMRC version, as it will have been created to meet specific needs and context within HMRC.
+</p>
+
+<p class="govuk-body">
+Designers should use these libraries for internal services where possible. When prototyping services for internal users, or services not on GOV.UK, designers should follow the relevant brand and typography guidance for their platform. 
+</p>
+
+<h3 class="govuk-heading-m" id="Get-support">
+  Get support
+</h3>
+
+<p class="govuk-body">
+Designers can get support with HMRC-specific patterns from the HMRC Design Resources team on HMRC Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#" class="govuk-link">#team-design-system</a>. Service patterns also have a link to the discussions on <a href="https://github.com/hmrc/design-patterns/issues" class="govuk-link">GitHub</a>.
+</p>
+
+<p class="govuk-body">
+For wider government patterns and components, the GOV.UK Design System Team is available on cross-government Slack at <a href="https://hmrcdigital.slack.com/archives/C014S77JCJW?name=C014S77JCJW#">#team-design-system</a>.
+</p>
+
+<h3 class="govuk-heading-m" id="using-prototypes">
+Using prototypes
+</h3>
+
+<p class="govuk-body">
+At HMRC we use a mix of <a href="https://companieshouse.blog.gov.uk/2023/09/13/how-prototyping-helps-us-deliver-better-user-experiences/" class="govuk-link">high and low-fidelity prototypes</a> to explain and test our design ideas. 
+</p>
+    
+<p class="govuk-body">
+Prototypes usually get more detailed as the service progresses through <a href="https://www.gov.uk/service-manual/agile-delivery" class="govuk-link">delivery phases</a>. 
+</p>
+    
+<p class="govuk-body">
+Low-fidelity prototypes are typically used in discovery and alpha, with high-fidelity prototyping being more common from alpha to live.
+</p>
+
+<h2 class="govuk-heading-l" id="Low-fidelity-prototypes">
+Low-fidelity prototypes
+</h2>
+
+<p class="govuk-body">
+HMRC’s preferred approach is to design using low-fidelity prototypes in early project stages.
+</p>
+    
+<p class="govuk-body">
+Designers can use low-fidelity prototypes to explore ideas quickly and cheaply before creating ‘high-fidelity’ prototypes that look more like the final product. Low-fidelity prototypes should be used in addition to high-fidelity prototypes - not instead of them.
```

Comment:
There's a mixture of cases 'Low-fidelity' & 'low-fidelity'. Is on of them correct or are they both correct depending on how you're using them?